src/components/DailyWeather/DailyWeatherDetails.tsx

```diff
@@ -58,11 +59,22 @@ const ArrowRightIconWrapperRestyled = styled(ArrowRightIconWrapper)`
   border-radius: 0 var(--border-radius) var(--border-radius) 0;
 `;
 
+const WindDirectionWrapper = styled.span`
+  position: relative;
+`;
+
+const WindDirectionIconStyled = styled(WindDirectionIcon)`
+  position: absolute;
+  transform: ${({ deg }) => `rotate(${deg}deg)`};
+  height: 1.5em;
+  right: 0.2em;
+`;
```

Comment:
Kinda odd way of placing inline icon, but let's do that in future.

Reply:
it was hard to place this icon in the right place